tests/unit/controller/eme-controller.js

```diff
@@ -118,6 +120,8 @@ describe('EMEController', function () {
   });
 
   it('should require all config functions when EME is enabled', function () {
+    ensureHttps(this);
```

Comment:
@gregrealeyes Is this here because beforeEach doesn't give a reference to the test? Otherwise it seems like beforeEach would be the perfect place to do this.

Reply:
agreed

Reply:
I have to do it here because the functions passed to the `it` calls are the only things that get executed in the browser. If I do it once in the `describe` call, `window` is not available